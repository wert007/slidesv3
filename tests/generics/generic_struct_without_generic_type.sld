struct A<Type> {
    a: int;
}

func main() {
    let a = new A(3);
}