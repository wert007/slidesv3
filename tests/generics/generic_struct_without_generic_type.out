Error in ..\tests\generics\generic_struct_without_generic_type.sld:1:8: Declared struct A as generic, but generic type $Type was not used. Maybe you forgot it?
